lib/msal-common/src/request/RefreshTokenRequest.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @type RefreshTokenRequest
+ *
+ * scopes:                  A space-separated array of scopes for the same resource.
+ * authority:               URL of the authority, the security token service (STS) from which MSAL will acquire tokens.
+ * refreshToken:            A refresh token returned from a previous request to the Identity provider.
+ */
+export class RefreshTokenRequest {
+    scopes: Array<string>;
+    refreshToken: string;
```

Comment:
I thought the refreshToken wasn't going to be returned to users?

Reply:
Will not be returned. But people using ADAL will have refresh tokens, and this will be used for migrating them to MSAL. (They will only use this API once, and then cache will be populated, so they can continue using acquire token silent thereafter). This is documented in the comment above the method in the code, and will also be part of some migration document that will have to be made available to people.